evan_did_method_spec.md

```diff
@@ -31,30 +31,33 @@ evan identifiers are lowercased hex-encoded strings. There are two types of iden
 An evan.network DID is depending on the type of the identity. There are three types of identities:<br>
 
 **Self-sovereign Identity (SSI)**<br>
-Self-sovereign identities (SSIs) can either be a company, a device, or a user. These identities are created, owned, and managed by a person via an externally owned account/private key and have their own means of authentication and signing and can issue verified credentials. Changes to the DID document itself can only be performed by the owner or another SSI referenced via the `controller` property of the DID document. Upon creation the `controller` property is set to the owner of the identity so that the owner has the full controll over the identity. SSIs fully feature key revocation and key rotation.
+Self-sovereign identities (SSIs) can either be a company, a device, or a user. These identities are created, owned, and managed by a person via an externally owned account/private key and have their own means of authentication and signing and can issue verified credentials. Changes to the DID document itself can only be performed by the owner or another SSI referenced via the `controller` property of the DID document. Upon creation the `controller` property is set to the owner of the identity so that the owner has the full control over the identity. Alongside this, each DID document holds a proof which is generated using the owner's key. SSIs fully feature key revocation and key rotation.
```

Comment:
```suggestion
Self-sovereign identities (SSIs) can either be a company, a device, or a user. These identities are created, owned, and managed by a person via an externally owned account/private key and have their own means of authentication and signing and can issue verifiable credentials (VCs). Changes to the DID document itself can only be performed by the owner or another SSI referenced via the `controller` property of the DID document. Upon creation the `controller` property is set to the owner of the identity so that the owner has the full control over the identity. Alongside this, each DID document holds a proof which is generated using the owner's key. SSIs fully feature key revocation and key rotation.
```

Reply:
Dropped the "Upon creation the `controller` property is set to the owner..." because that is not the case for us

Reply:
>  These identities are created, owned, and managed by a person via an externally owned account/private key and have their own means of authentication and signing and can issue verifiable credentials (VCs)

This sentence has a lot of 'and's which make it hard to understand. Maybe we can split that in 2 or 3 short sentences or steps. See also: http://www.cypressmedia.net/articles/article/26/six_principles_of_technical_writing#sentences